cosmo_tester/framework/cluster.py

```diff
@@ -609,8 +613,11 @@ class BootstrapBasedCloudifyCluster(CloudifyCluster):
 
     def __init__(self, *args, **kwargs):
         super(BootstrapBasedCloudifyCluster, self).__init__(*args, **kwargs)
-        self._manager_resources_package = \
-            util.get_manager_resources_package_url()
+        if util.is_community():
+            self._manager_resources_package = None
```

Comment:
I'll put a comment in, but essentially it's because the built-in resources package (the default) should be a community one, unless we're making our manager blueprints deliberately unusable.